backend/db/migrate/20220817011608_create_property_assessments.rb

```diff
@@ -1,69 +1,77 @@
 # frozen_string_literal: true
 
+# rubocop:disable Metrics/AbcSize, Metrics/BlockLength, Metrics/MethodLength
 class CreatePropertyAssessments < ActiveRecord::Migration[7.0]
   def change
+    # Schema for Boston Property Assessment Data.
+    # Data key: https://data.boston.gov/dataset/property-assessment/resource/db9d1e04-080b-4477-96e0-c440f3ff29f1
     create_table :property_assessments do |t|
       t.timestamps
-      t.string :heat_type
-      t.string :heat_fuel
-      t.string :ac_type
-      t.integer :fire_place
-      t.string :orientation
-      t.integer :num_parking
-      t.string :prop_view
-      t.string :corner_unit
-      t.integer :com_units
-      t.integer :rc_units
-      t.integer :land_sf
-      t.integer :gross_area
-      t.integer :living_area
-      t.integer :land_value
-      t.integer :bldg_value
-      t.integer :total_value
-
       t.integer :pid
       t.integer :cm_id
       t.integer :gis_id
-      t.string :st_num
-      t.string :st_name
+      t.string  :st_num
+      t.string  :st_name
       t.integer :unit_num
-      t.string :city
+      t.string  :city
       t.integer :zipcode
-
       t.integer :bldg_seq
       t.integer :num_bldgs
       t.integer :luc
-      t.string :luc_desc
-      t.string :bldg_type
-      t.string :own_occ # no char datatype?
-
-      t.string :owner
-      t.string :mail_addressee
-      t.string :mail_address
-      t.string :mail_city
-      t.string :mail_state, limit: 2
-      t.string :mail_zipcode # these can be 5 or zip+4 format
-      t.float :res_floor
+      t.string  :lu, limit: 2
+      t.string  :luc_desc
```

Comment:
```suggestion
      t.string  :lu_desc
```

Typo I think - it's lu_desc in the spreadsheet